backend/models.py

```diff
@@ -202,10 +201,8 @@ def get(self, model):
         else:
             schema = list(model.fields_get().keys())
         schema = unflatdot(schema)
-        items = []
-        for item_id in res_ids:
-            values = model.read(item_id, list(schema.keys()))
-            items.append(normalize(model, values, schema))
+        fields = list(schema.keys())
+        items = model.read(search_params, fields=fields, limit=limit, offset=offset)
```

Comment:
You have removed `normalize()` call and I think it make something that can't be omitted

Reply:
@ecarreras that's true, I did another commit normalizing the results after reading